app/views/organisations/_corporate_information.html.erb

```diff
@@ -1,16 +1,20 @@
-<div class="column-one-third">
+<div class="column-one-third organisation__float-section">
   <% if @show.corporate_information[:corporate_information_links][:items].any? %>
-    <%= render "govuk_publishing_components/components/heading", {
-      text: t('organisations.corporate_information'),
-      margin_bottom: 2,
-      padding: true,
-      border_top: 5,
-      brand: @organisation.brand,
-    } %>
-    <%= render "components/topic-list", @show.corporate_information[:corporate_information_links] %>
+    <% unless @organisation.is_promotional_org? %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: t('organisations.corporate_information'),
+        margin_bottom: 2,
+        padding: true,
+        border_top: 5,
+        brand: @organisation.brand
+      } %>
+    <% end %>
+    <div class="<%= "organisation__section-border-top" if @organisation.is_promotional_org? %>">
```

Comment:
Should we be using the branding classes (used elsewhere) here? I note the additional padding so maybe we still need a class to add that as well.